unison-src/transcripts/resolve.output.md

```diff
@@ -194,21 +194,22 @@ The namespace `c` now has an edit conflict, since the term `foo` was edited in t
   have been merged into this one. You'll have to tell me what to
   use as the new definition:
   
-    The term #44954ulpdf was replaced with foo#8e68dvpr0a and
-    foo#jdqoenu794
+    The term 1. #44954ulpdf was replaced with
```

Comment:
I don't love inlining the number into a sentence like this (but it's not the worst thing I guess 🤷🏼‍♂️ ) , anyone have better suggestions?

Reply:
The following isn't great either, but is it better?
```
1. The term #44954 was replaced with
    2. foo#8e68
    3. foo#jdqo

Reply:
Another option is making the color half invisible or not displaying the number, while counting it anyway, and you have to subtract one from the 2 to get it ;-)

Reply:
> is it better?

I don't have too much of a preference between the two, this might just be a little awkward and that's probably okay 👍🏼